Pokepedia.ApiAdapter/Models/PokemonModel.cs

```diff
@@ -6,5 +6,7 @@ public class PokemonModel
         public int Id { get; set; } = 0;
         public int Weight { get; set; } = 0;
         public int Order { get; set; } = 0;
+        public string LocationArea { get; set; } = string.Empty;
```

Comment:
I dont love this but I also cant think of a better idea currently so i guess it can stay

Reply:
I change my mind. This should be a Location Model. (for now at least)